src/test/java/com/example/training/controllers/UserControllerTest.java

```diff
@@ -28,7 +32,9 @@
     private static final String SINGLE_USER_PATH = "src/test/resources/user/singleUser.json";
     private static final User TEST_USER1 = new User("2a3993ee-d74e-42ac-af90-9390642db803", "John", "Doe");
     private static final User TEST_USER2 = new User("8644d523-eaf2-412b-87b9-82ceba72ec6b", "Foo", "Bar");
+    private static final User TEST_USER_TO_ADD = new User("8644d523-eaf2-412b-87349-82ceba72ec6b", "Joe", "Ziggy");
     private MockMvc mockMvc;
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
move OBJECT_MAPPER before mockMvc.
private static final should be before private